internal/util/debug/archive.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package debug provides debug facilities.
+package debug
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"path/filepath"
+)
+
+type archiveUrlList []url.URL
+
+// archiveHandler - is a handler for creating the archive containing all the debug and metrics info.
```

Comment:
```suggestion
// archiveHandler is a handler for creating the archive containing all the debug and metrics info.
```

To follow all other declaration comments.

But that comment is also incorrect – that archive does not contain all debug and metrics info.

Reply:
Updated the sentence. This was added when i was adding all the files to zip.